Task #2 - Fixed description issues and adds proper references

daahl opened the pull request:
- Fix for short and long description issues. Some records were improperly split between short and long descriptions. This was fixed by using regex for splitting instead of string splitting.
    - See record _separ -> FAC2SCR_
- Replace references using aux file and adding “… from the manual”.
    - See records _damage -> LRCOIL_ and _output -> ITRA_ for examples

hobler commented:
Please use forward slash (/) as a path separator. Should work for Linux+Windows, while backslash (\\) works only for Windows. See, e.g., line 562 of parsing.py.

daahl commented:
I've replaced the back slashes with forward slashes and pushed the changes to this PR.

hobler commented:
Could you please fix the following items:

- In LaTeX formulas, blanks only serve to separate commands and/or text. They should be removed during parsing. E.g. in the long description of `&DAMAGE/FPOCK` the transformed formula contains a lot of whitespace that should be eliminated.
- Any text following `%` on the same line should be ignored (see `&MATERIAL/DENSITY`)
- Any itemization (in long description, range, etc) should not add a `-` or anything else, since a new line not always means a new item (e.g. in `ESURF`). Also, when a new line starts with a number, it reads as the negative of the number with the `-` symbol, which is misleading.
- `\min` in a formula should be translated to `min`, not `\\min` (see `PMAXPSI`).

In addition, if that's not too complicated, can you implement a translation of `\sqrt[b]{a}` to `^b \sqrt{a}` followed by the existing translation of `^b \sqrt{a}`? And/or a translation of `a^{b/c}` to `^c \sqrt{a^b}?

daahl commented:
Added the requested items including the function to reformat square roots.

hobler commented:
Regarding the requested items: Thanks for adding them. Just the removal of text following `%` doesn't seem to work (see `long_desc` of `&MATERIAL/DENSITY`.

Regarding coding style: Overall looks good. 

- Some lines are longer than 80 characters. Since one should adhere to existing conventions in a code base, please fix this.
- In some docstrings the listed function parameters do not match the actual parameters.
- Docstrings sometimes refer to reStructuredText. This stems from the earlier version. Should be replaced by "Unicode".

daahl commented:
Fixed the issues now. I removed the remove_comments() function completely, and instead made use of the line iteration that's already happening at the start of the parsing.

hobler commented:
There are still references to reStructuredText (mostly to reST), also in one function name.

daahl commented:
Wow, not sure how I could have missed those... I replaced them now and an additional one that was in manual2db.
